rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE.
 /// \param new_order
 /// \param is_output
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalSequenceRearrange(RocalContext p_context, RocalTensor p_input,
+extern "C" RocalTensor  ROCAL_API_CALL rocalSequenceRearrange(RocalContext p_context, RocalTensor p_input,
```

Comment:
@fiona-gladwin All changes to header files should have comments like the ones we have added to vx_rpp extension. Please follow the same format as rocAL repo. All the changes implemented there for the documentation needs to be merged to this branch

Reply:
Done

Reply:
@fiona-gladwin I do not think it follows the format. Please refer to this PR. The inputs need to be ```[in]``` and so on. Please also merge with the TOT on rocAL repo, since that has the code needed to generate the files. Once it's merged we have to run that one line command to generate the file to make sure it looks ok. Please let me know if you have any questions

Reply:
@LakshmiKumar23 In rocAL repo this is the format that is added for the comments hence we followed the same - [link](https://github.com/ROCmSoftwarePlatform/rocAL/blob/516cdc2e4738635487037c4e88f4434fd84281b7/rocAL/include/api/rocal_api_augmentation.h#L173)

Please let me know if we need to have comments similar to vx_rpp or rocAL repo and we can make changes accordingly

Reply:
@fiona-gladwin https://github.com/ROCmSoftwarePlatform/rocAL/commit/6ed79977410f3a6bb03fff9972f0de52c6c7e39b#diff-3ee61a464ddcf9a70dd656c57a8af4355a28eb8820c1acaea25b153542f99109
like in this PR.

Reply:
@LakshmiKumar23 In the above link I could see in rocal_api_augmentations.h the [in] and [out] specifiers are not present.
Should we add them?

Reply:
@fiona-gladwin Sorry, wrong link https://github.com/ROCmSoftwarePlatform/rocAL/commit/6ed79977410f3a6bb03fff9972f0de52c6c7e39b#diff-d1172c0cf6822f490cd3502421e798f0fd01dd040bb8b3285df37ea92fb836eb